sources/packages/web/src/components/common/students/StudentScholasticStandingLimitedHistory.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header title="Scholastic Standing Limited History" />
+    </template>
+    <content-group
+      ><p class="label-bold-normal">
+        Number of Lifetime Weeks of Unsuccessful Completion:
+        {{ scholasticStandingSummary.lifetimeUnsuccessfulCompletionWeeks }}
+      </p></content-group
+    >
+  </body-header-container>
+</template>
+<script lang="ts">
+import { ScholasticStandingSummaryDetailsAPIOutDTO } from "@/services/http/dto";
+import { ScholasticStandingService } from "@/services/ScholasticStandingService";
+import { defineComponent, onMounted, ref } from "vue";
+
+export default defineComponent({
+  props: {
+    studentId: {
+      type: Number,
+      required: false,
```

Comment:
Can it potentially be null at some point? If yes we would probably need a watchEffect instead onMount, right?